lib/puppet/type/cisco_command_config.rb

```diff
@@ -106,4 +106,37 @@ def self.title_patterns
       value.gsub!(/^(#{indent_level})/, '') # remove extra indentation
     end # validate
   end # property command
+
+  newproperty(:test_get) do
+    desc %(
+      This is a test-only property for beaker use. It allows beaker to retrieve
+      any configuration it needs from the device using puppet resource. Callers
+      must pass a filter string to test_get.
+      Example usage:
+        puppet resource cisco_command_config 'cc' test_get='incl feature'
+    )
+
+    def insync?(*)
+      # This is a "get-only" property so insync? is overridden to prevent
+      # puppet from displaying the following notice:
+      #   Notice: /Cisco_command_config[c]/test_get: test_get changed '' to ''
+      true
```

Comment:
It's the idempotence checker method so I just deduced that always returning true would avoid the Notice.